azure-mgmt-resources/src/main/java/com/azure/management/AzureTokenCredential.java

```diff
@@ -43,33 +43,33 @@ public AzureTokenCredential(AzureEnvironment environment, String domain) {
      * @param request the details of the token request
      * @return a Publisher that emits a single access token
      */
-    @Override
-    public final Mono<AccessToken> getToken(TokenRequestContext request) {

```

Comment:
OK for now, please investigate how other implementation of this class do on this `getToken`